src/commands/emergency_solution.rs

```diff
@@ -45,5 +47,51 @@ where
 		+ 'static,
 	T::Solution: Send,
 {
-	todo!("not possible to implement yet");
+	let round = api
+		.storage()
+		.at(config.at)
+		.await?
+		.fetch_or_default(&runtime::storage().election_provider_multi_phase().round())
+		.await?;
+
+	let miner_solution = epm::fetch_snapshot_and_mine_solution::<T>(
+		&api,
+		config.at,
+		config.solver,
+		round,
+		config.force_winner_count,
+	)
+	.await?;
+
+	let ready_solution = miner_solution.feasibility_check()?;
+	let encoded_size = ready_solution.encoded_size();
+	let score = ready_solution.score;
+
+	let mut supports = ready_solution.supports.into_inner();
+
+	// maybe truncate.
+	if let Some(force_winner_count) = config.force_winner_count {
+		log::info!(
+			target: LOG_TARGET,
+			"truncating {} winners to {}",
+			supports.len(),
+			force_winner_count
+		);
+		supports.sort_unstable_by_key(|(_, s)| s.total);
+		supports.truncate(force_winner_count as usize);
+	}
+
+	// write to file and stdout.
+	let encoded_supports = supports.encode();
+	let mut supports_file = std::fs::File::create("solution.supports.bin")?;
+	supports_file.write_all(&encoded_supports)?;
+
+	log::info!(target: LOG_TARGET, "ReadySolution: size {:?} / score = {:?}", encoded_size, score);
+	log::trace!(
+		target: LOG_TARGET,
+		"Supports: {}",
```

Comment:
Current code would [truncate](https://github.com/paritytech/substrate/blob/master/primitives/core/src/hexdisplay.rs#L40) the hex display. Should we use debug display? 

I also made this [small PR](https://github.com/paritytech/polkadot/pull/7194/files) for miner v1 printing encoded call instead of `supports`.

```suggestion
		"Supports: {:?}",
```

Reply:
Might be better to save it in a file as well.